### ISSUE TITLE
Cannot build Chrome extensions with Node 0.12

### ISSUE THREAD

**Issue by compwron** [malept/octoaudit#6]
I was trying to build locally because I want to use the github enterprise plugin. 

````
✔ data/img/octoaudit-16.png (scale: 0.0625)
✔ data/img/octoaudit-48.png (scale: 0.1875)
✔ data/img/octoaudit-128.png (scale: 0.5)
✔ data/img/octoaudit-64.png (scale: 0.25)
✔ data/img/octoaudit-256.png (scale: 1)

Running "copy:octoaudit" (copy) task
Created 3 directories, copied 15 files

Running "copy:octoaudit_enterprise" (copy) task
Created 3 directories, copied 15 files

Running "copy:xpi" (copy) task
Created 3 directories, copied 11 files

Running "template:octoaudit" (template) task
File `build/octoaudit/manifest.json` created.

Running "template:octoaudit_enterprise" (template) task
File `build/octoaudit_enterprise/manifest.json` created.

Running "mozilla-addon-sdk:1_17" (mozilla-addon-sdk) task
Mozilla Addon SDK already downloaded

Running "mozilla-cfx-xpi:xpi" (mozilla-cfx-xpi) task
Creating dist dir '/Users/me/bc/octoaudit/dist/xpi'...
Creating xpi...
Generated XPI: /Users/me/bc/octoaudit/dist/xpi/octoaudit.xpi

Running "crx:octoaudit" (crx) task
Fatal error: Cannot read property 'length' of null
TypeError: Cannot read property 'length' of null
  at Function.Buffer.concat (buffer.js:189:24)
  at [object Object].<anonymous> (/Users/me/bc/octoaudit/node_modules/crx/src/crx.js:130:102)
  at [object Object].emit (events.js:104:17)
  at emitReadable_ (_stream_readable.js:424:10)
  at emitReadable (_stream_readable.js:418:7)
  at onEofChunk (_stream_readable.js:401:3)
  at readableAddChunk (_stream_readable.js:146:7)
  at [object Object].Readable.push (_stream_readable.js:126:10)
  at [object Object].Transform.push (_stream_transform.js:140:32)
  at done (_stream_transform.js:203:17)
  at [object Object].<anonymous> (_stream_transform.js:134:7)
  at [object Object].g (events.js:199:16)
  at [object Object].emit (events.js:104:17)
  at prefinish (_stream_writable.js:474:12)
  at finishMaybe (_stream_writable.js:482:7)
  at endWritable (_stream_writable.js:493:3)
  at [object Object].Writable.end (_stream_writable.js:459:5)
  at [object Object].onend (/Users/me/bc/octoaudit/node_modules/crx/node_modules/archiver/node_modules/readable-stream/lib/_stream_readable.js:523:10)
  at [object Object].g (events.js:199:16)
  at [object Object].emit (events.js:129:20)
  at /Users/me/bc/octoaudit/node_modules/crx/node_modules/archiver/node_modules/readable-stream/lib/_stream_readable.js:965:16
  at process._tickCallback (node.js:355:11)

GM20437:~/bc/octoaudit$ 
GM20437:~/bc/octoaudit$ ls ~/.ssh/chrome-apps.pem
/Users/me/.ssh/chrome-apps.pem
GM20437:~/bc/octoaudit$
````

**Comment by malept** [comment]
I assume since your pem path is `/Users`, you're running OSX? If so, which version of OSX is it? Also, which version of Node/io.js is this? (Any other relevant data about your environment would also be helpful.)

**Comment by malept** [comment]
I cannot reproduce this error on my Debian machine, running Node 0.10.29.

**Comment by malept** [comment]
I have reproduced this issue in [Travis CI](https://travis-ci.org/malept/octoaudit/jobs/69307563) - it appears that you are using Node 0.12.x.

**Comment by malept** [comment]
I fixed this by upgrading the CRX tooling, see 83aa0a9ae9c8ff6a0c6c057939cfeebe0cd48c8a (and associated [Travis build](https://travis-ci.org/malept/octoaudit/jobs/69314926)). Thanks for reporting!

**Comment by compwron** [comment]
Sorry for my late reply. Thank you so much for helping me out! I pulled and rebuilt and it worked great. 

You are correct that I am using node 0.12. I know that this is bad but there are other reasons for it. :/ 
````
GM20437:~/bc/octoaudit$ node -v
v0.12.0
````

![screen shot 2015-07-02 at 10 40 53 am](https://cloud.githubusercontent.com/assets/578159/8483644/d87e0816-20a6-11e5-9ab4-337e057cb414.png)
